safe_control_gym/envs/gym_control/cartpole.py

```diff
@@ -438,12 +438,16 @@ def _set_action_space(self):
 
     def _set_observation_space(self):
         '''Sets the observation space of the environment.'''
-        # Angle at which to fail the episode.
-        self.theta_threshold_radians = 90 * math.pi / 180
         # NOTE: different value in PyBullet gym (0.4) and OpenAI gym (2.4).
         self.x_threshold = 2.4
+        self.x_dot_threshold = 10
+        self.theta_threshold_radians = 90 * math.pi / 180  # Angle at which to fail the episode.
+        self.theta_dot_threshold = 10
         # Limit set to 2x: i.e. a failing observation is still within bounds.
```

Comment:
That was actually me. I added this purely for acados since acados cannot handle a inf large constraint. https://discourse.acados.org/t/infeasible-qps-when-using-nonlinear-casadi-constraint-expressions/1595/3?u=mxche

Reply:
Sounds good let me know what you decide

Reply:
I will double them for now, as well as for the quadrotor.